vcd/org_vdc_shared_test.go

```diff
@@ -0,0 +1,335 @@
+// +build vdc nsxt ALL functional
```

Comment:
maybe is better to rename file to something like `vdc_util_test.go` `vdc_common_test.go` to avoid opening/read this as normal/running test as others are with same naming convention.

Reply:
Renamed to `vdc_common_test.go`